frontend/src/api/auth.js

```diff
@@ -0,0 +1,29 @@
+import fetcher from './helpers';
+
+const baseUrl = '/api/auth';
+
+const loginUser = ({ email, password }) =>
```

Comment:
Nitpick: if a `body`(or called `data` or something else) prop were introduced to this payload and `email` and `password` were a part of it, then there would be no need to destructure here and then create a new object before stringifying it. An added benefit is if there is a need to start sending additional info, there would be no need to update this method since it would all be in the `body` object.
And the call to JSON stringify could be moved to the `fetcher` wrapper since I assume all calls to it would do it. At least the ones with the `body` prop.

Reply:
sure, all params can be sent directly without destructuring, but in my opinion this provides an additional layer of documentation for someone that wants to use this method